packages/qdrant-loader/tests/scripts/check_worker_pool.py

```
@@ -0,0 +1,110 @@
+            active_by_source[source] = now_active_src
+            max_by_source[source] = max(max_by_source.get(source, 0), now_active_src)
+
+        LOG.info(
+            "START job_type=%s source=%s seq=%s active_total=%s active_source=%s",
+            job_type, source, seq, active_total, active_by_source[source]
+        )
+
+        await asyncio.sleep(0.05)
+
+        if args.fail_every > 0 and seq % args.fail_every == 0:
+            raise RuntimeError(f"forced failure for seq={seq}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)